src/components/profile/profile-logged-in/profile-details/delete-account-dialog.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useCallback } from "react";
+import { useAuthStore } from "../../../../auth/auth-store.tsx";
+import { useErrorStore } from "../../../../error/error-store.tsx";
+import { useUrlState } from "../../../router/store.tsx";
+import { useI18nStore } from "../../../../i18n/i18n-store.ts";
+import { SecondaryDestructiveButton } from "../../../buttons/secondary-destructive.tsx";
+import { PrimaryButton } from "../../../buttons/primary.tsx";
+
+export const DeleteAccountDialog: React.FC = () => {
+	const i18n = useI18nStore().i18n();
+	const { deleteUser } = useAuthStore();
+	const { handleError } = useErrorStore();
+	const { setSearchParams } = useUrlState();
+
+	const onConfirm = useCallback(async () => {
+		try {
+			await deleteUser();
+			setSearchParams(new URLSearchParams());
+		} catch (error) {
+			handleError(i18n.common.defaultErrorMessage, error);
+		}
+	}, []);
+
+	return (
+		<dialog
+			id={"delete-account-dialog"}
+			className="pt-9 pb-12 px-14 bg-white shadow rounded-2xl backdrop:backdrop-blur"
```

Comment:
there is a bit too much white space with these padding settings. I checked again on figma, and I would change it to: pt-10 pb-8 px-12

Reply:
will do 👍